codegenerator/cli/templates/static/codegen/src/Benchmark.res

```diff
@@ -187,8 +261,7 @@ module Summary = {
     @as("std-dev") stdDev: float,
```

Comment:
Never worked with std-dev before. I usually look at mean and avg

Reply:
I use it quite a lot. It will essentially give you an idea of how much variance your data has.
![Screenshot 2024-10-28 at 09 33 56](https://github.com/user-attachments/assets/c0c64fd8-2e32-4071-bc00-f070a5148341)
For eg. the circle in green has low variance and the mean should be an accurate representation. 

The red circle has high variance so something makes it inconsistent. Could speak to something that is cached vs not-cached on different runs etc.